nix/cardano-services/deployments/backend-ingress.nix

```
@@ -63,6 +63,16 @@
                 }
               )
               values.backend.routes)
+            ++ lib.optionals config.providers.chain-history-provider.enabled [
+              {
+                pathType = "Prefix";
+                path = "/v${lib.last (lib.sort lib.versionOlder values.cardano-services.versions.chainHistory)}/chain-history";
+                backend.service = {
+                  name = "${chart.name}-chain-history-provider";
+                  port.name = "http";
+                };
+              }
+            ]
```

```suggestion
            ++ lib.optionals config.providers.chain-history-provider.enabled (
              (version: {
                pathType = "Prefix";
                path = "/v${version}/chain-history";
                backend.service = {
                  name = "${chart.name}-chain-history-provider";
                  port.name = "http";
                };
              }) values.cardano-services.versions.chainHistory
            )
```

also can you run treefmt?

---

I do not understand the comment, when I try to use the suggested "(version:" my working deploy fails, will leave it like this for now

---

@bernokl I think there's a `map` missing in the suggestion from @gytis-ivaskevicius, so instead of:
```
((version: { … }) values.⟨…⟩.chainHistory)
```
it should be:
```
(map (version: { … }) values.⟨…⟩.chainHistory)
```